galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -172,6 +172,7 @@ var testGrid = []testCase{
 		expected: []message{
 			{msg.ReferencedResourceNotFound, "VirtualService reviews-bogushost.default"},
 			{msg.ReferencedResourceNotFound, "VirtualService reviews-bookinfo-other.default"},
+			{msg.ReferencedResourceNotFound, "VirtualService reviews-mirror-bogushost.default"},
```

Comment:
Would it be useful to have a test case where the mirrored host exists and we ensure no message is emitted? If you think there's no possible way such a test might fail, then I'm fine leaving this as is.

Reply:
It's probably overkill but it's also cheap, I'll do it.